publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -276,6 +293,9 @@ private void initService() {
         if (channelsCount == null ) {
             channelsCount = Integer.getInteger(getValuesFromSystemProperties(protocol + ".rabbitmq.channelsCount"));
         }
+        if (waitForConfirmsTimeOut == null ) {
+            waitForConfirmsTimeOut = Long.getLong(getValuesFromSystemProperties(protocol + ".rabbitmq.waitForConfirmsTimeOut"));
```

Comment:
What if getLong fails? Either we should use a default value or don't use confirm functionality at all. Analogously for line 294.

Reply:
if getLongfails and still the value of waitForConfirmsTimeout is null then it uses default value